<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Генератор рандомизированной пирамиды поиска</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QGraphicsView" name="outputGraphicsView"/>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="outputLineEdit">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Место для элементов в порядке возрастания</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="calculatePushButton">
      <property name="text">
       <string>Запуск</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="inputLineEdit">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Место для исходных данных</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Дополнительно</string>
    </property>
    <addaction name="downloadFromFileAction"/>
    <addaction name="generationWindowAction"/>
    <addaction name="helpAction"/>
    <addaction name="authorInformationAction"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="downloadFromFileAction">
   <property name="text">
    <string>Загрузить данные из файла</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
  <action name="authorInformationAction">
   <property name="text">
    <string>Информация об авторе</string>
   </property>
  </action>
  <action name="generationWindowAction">
   <property name="text">
    <string>Текущий контроль</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
